lib/screens/creator_hub/widgets/draft_list_tile.dart

```diff
@@ -63,49 +64,47 @@ class DraftListTile extends StatelessWidget {
+                          child: SvgPicture.asset(
+                            kSvgNftFormatImage,
+                            color: EaselAppTheme.kBlack,
+                          ),
                         ),
+                        placeholder: (context, url) => Shimmer(color: EaselAppTheme.cardBackground, child: const SizedBox.expand()),
+                        imageUrl: nft.url.changeDomain(),
+                        fit: BoxFit.cover,
                       ),
-                      placeholder: (context, url) => Shimmer(color: EaselAppTheme.cardBackground, child: const SizedBox.expand()),
-                      imageUrl:  nft.url.changeDomain(),
-                      fit: BoxFit.cover,
-                    ),
-                    onVideo: (context) => CachedNetworkImage(
-                      fit: BoxFit.fill,
-                      imageUrl: nft.thumbnailUrl.changeDomain(),
-                      errorWidget: (a, b, c) => const Center(child: Icon(Icons.error_outline)),
-                      placeholder: (context, url) => Shimmer(color: EaselAppTheme.cardBackground, child: const SizedBox.expand()),
-                    ),
-                    onPdf: (context) => CachedNetworkImage(
-                      fit: BoxFit.fill,
-                      imageUrl: nft.thumbnailUrl,
-                      errorWidget: (a, b, c) => const Center(child: Icon(Icons.error_outline)),
-                      placeholder: (context, url) => Shimmer(color: EaselAppTheme.cardBackground, child: const SizedBox.expand()),
-                    ),
-                    onAudio: (context) => CachedNetworkImage(
-                      fit: BoxFit.fill,
-                      imageUrl: nft.thumbnailUrl.changeDomain(),
-                      errorWidget: (a, b, c) => const Center(child: Icon(Icons.error_outline)),
-                      placeholder: (context, url) => Shimmer(color: EaselAppTheme.cardBackground, child: const SizedBox.expand()),
-                    ),
-                    on3D: (context) => ModelViewer(
-                      src: nft.url.changeDomain(),
-                      backgroundColor: EaselAppTheme.kWhite,
-                      ar: false,
-                      autoRotate: false,
-                      cameraControls: false,
-                    ),
-                    assetType: nft.assetType.toAssetTypeEnum(),
-                  )
-                ),
+                      onVideo: (context) => CachedNetworkImage(
+                        fit: BoxFit.fill,
+                        imageUrl: nft.thumbnailUrl.changeDomain(),
+                        errorWidget: (a, b, c) => const Center(child: Icon(Icons.error_outline)),
+                        placeholder: (context, url) => Shimmer(color: EaselAppTheme.cardBackground, child: const SizedBox.expand()),
+                      ),
+                      onPdf: (context) => CachedNetworkImage(
+                        fit: BoxFit.fill,
```

Comment:
All three cases are resolving into the same code block, reusable code would be better.